Added new models + updated existing models from API reference

jeff-99 opened the pull request:
I've added all new models and updated the old models using the tampermonkey script

stephangroen commented:
Thank you @jeff-99 

Could you add the new models to the test file: https://github.com/picqer/exact-php-client/blob/master/tests/EntityTest.php

Also some Models got a double point now after the class name. Could you fix that?

jeff-99 commented:
Hi @stephangroen , 

I've added the test cases and I refactored the performEntityTest method to check if a model is a subclass of the Model class instead of checking if the direct parent is the Model class

jeff-99 commented:
Hi @stephangroen, 

I would like to add some namespace seperation for all models and would love to get your feedback.
My preference would be to copy the structure of the API docs by moving the models to a "Service" namespace. 

This would result in something like the list below: 
* Picqer\Financials\Exact\Accountancy
	* Picqer\Financials\Exact\Accountancy\AccountInvolvedAccount
	* Picqer\Financials\Exact\Accountancy\AccountOwner
	* etc.
* Picqer\Financials\Exact\Activities
	* Picqer\Financials\Exact\Activities\CommunicationNotes
	* etc.
* Picqer\Financials\Exact\Assets
* Picqer\Financials\Exact\Budget
* Picqer\Financials\Exact\Bulk
* Picqer\Financials\Exact\Cashflow
* Picqer\Financials\Exact\ContinuousMonitoring
* Picqer\Financials\Exact\CRM
* Picqer\Financials\Exact\Documents
* Picqer\Financials\Exact\Financial
* Picqer\Financials\Exact\FinancialTransaction
* Picqer\Financials\Exact\General
* Picqer\Financials\Exact\GeneralJournalEntry
* Picqer\Financials\Exact\HRM
* Picqer\Financials\Exact\Inventory
* Picqer\Financials\Exact\Logistics
* Picqer\Financials\Exact\Mailbox
* Picqer\Financials\Exact\Manufacturing
* Picqer\Financials\Exact\OpeningBalance
* Picqer\Financials\Exact\Payroll
* Picqer\Financials\Exact\Project
* Picqer\Financials\Exact\Purchase
* Picqer\Financials\Exact\PurchaseEntry
* Picqer\Financials\Exact\PurchaseOrder
* Picqer\Financials\Exact\Sales
* Picqer\Financials\Exact\SalesEntry
* Picqer\Financials\Exact\SalesInvoice
* Picqer\Financials\Exact\SalesOrder
* Picqer\Financials\Exact\Subscription
* Picqer\Financials\Exact\System
* Picqer\Financials\Exact\Users
* Picqer\Financials\Exact\VAT
* Picqer\Financials\Exact\Webhooks
* Picqer\Financials\Exact\Workflow

Is this something that you would like to include in this repository ?

stephangroen commented:
Can you fix the style issues?

alexandersch commented:
Can the conflicts be resolved so that this can be merged?

jeff-99 commented:
@stephangroen  @alexandersch  styling issues have been fixed.

vdlp-mw commented:
Any plans on when this will be merged? Thanks in advance 😄

stephangroen commented on src/Picqer/Financials/Exact/ReceivablesList.php:
```diff
@@ -3,34 +3,36 @@
  * @property string $YourRef Your reference
  */
-class ReceivableList extends Model
+class ReceivablesList extends Model
```
This is a breaking change without an alias.

stephangroen replied:
Will fix after merge.

stephangroen commented on src/Picqer/Financials/Exact/VatCode.php:
```diff
@@ -33,22 +33,23 @@
  * @property string $VATTransactionType Indicates the type of transactions for which the VAT code may be used. B = Both, P = Purchase, S = Sales
  */
-class VatCode extends Model
+class VATCode extends Model
```
This could break on case sensitive systems.

stephangroen commented:
Thanks @jeff-99 for the big effort!
